CHANGELOG.asciidoc

```diff
@@ -3,6 +3,17 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.11.1]]
+=== Beats version 8.11.1
+https://github.com/elastic/beats/compare/v8.11.0\...v8.11.1[View commits]
+
+==== Added
+
+*Affecting all Beats*
+
+- Setting environmental variable ELASTIC_NETINFO:false in Elastic Agent pod will disable the netinfo.enabled option of add_host_metadata processor
```

Comment:
```suggestion
- Setting environmental variable `ELASTIC_NETINFO:false` in {agent} pod will disable the `netinfo.enabled` option of add_host_metadata processor.
```
Just added backticks so that the variables appear in italic and we use the variable for "Elastic Agent", but of course this depends on Anderson's question above about whether or not we should mention Agent.